docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+services:
+    db:
+        image: mariadb
+        ports:
+        - "3306:3306"
+        environment:
+            MYSQL_ROOT_PASSWORD: prices123
+            MYSQL_DATABASE: aws_prices
+    importer:
+        build: ./importer
+        depends_on:
+        - db
+        links:
+        - db
+        environment:
+            MARIADB_USER: root
+            MARIADB_HOST: db
+        command: ["/scripts/wait-for", "db:3306", "--", "/scripts/pricing_import.py"]
+    server:
+        build: ./server
+        ports:
+        - "4000:4000"
+        depends_on:
+        - db
+        volumes:
+        - ./server/src/:/usr/src/app/src
```

Comment:
This worked for me locally from a fresh clone, although there still seems to be some caching. I'm not really sure why, but if it does `yarn install` in the volume directory it doesn't have access to `node_modules` or something even though the installation appears to succeed. Having a separate directory for the actual source to change seems to make a difference, although if you install other node modules I don't think that gets automatically reflected so would require a rebuilding of the container which is unfortunate. I think it may just be a limitation of Docker at this point, but I'm not sure.  See update to the [server/Dockerfile] as well to correspond to this.